index.html

```diff
@@ -71,15 +71,109 @@ <h2 class="small-name">Software Developer</h2>
+        <li class="social-it"><a href="#" class="fa fa-twitter fa-lg"></a></li>
+        <li class="social-it"><a href="#" class="fa fa-medium fa-lg"></a></li>
+         </ul>
+    
+     <div class="arrow-right"><img src="/img/right.png" alt=""></div>
+    </section>
+    <section class="my-works">
+        <p class="lets">My Recent Works</p>
+        <img src="img/circle-3.png" class="circle-3" alt="">
+        <img src="img/circle-4.png" class="circle-4" alt="">
+        <div class="all-items">
+        <div class="works item1"><img class="project-img" src="/img/project.png" alt=""></div>
+        <div class="works item2"><h1 class="work-name">Multi-Post Stories</h1>
+            <h2 class="work-des">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </h2>
+        <div class="work-lang">
+            <a href="">css</a><a href="">html</a><a href="">bootstrap</a><a href="">ruby</a>
+        </div>
+        <button>See Project</button>
+        </div>
+        <div class="works item3"><h1 class="work-name">Multi-Post Stories</h1>
+            <h2 class="work-des">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </h2>
+        <div class="work-lang">
+            <a href="">css</a><a href="">html</a><a href="">bootstrap</a><a href="">ruby</a>
+        </div>
+        <button>See Project</button>
+        </div>
+        <div class="works item4"><img class="project-img" src="/img/project.png" alt=""></div>
+        <div class="works item5"><img class="project-img" src="/img/project.png" alt=""></div>
+        <div class="works item6"><h1 class="work-name">Multi-Post Stories</h1>
+            <h2 class="work-des">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </h2>
+        <div class="work-lang">
+            <a href="">css</a><a href="">html</a><a href="">bootstrap</a><a href="">ruby</a>
+        </div>
+        <button>See Project</button>
+        </div>
+        <div class="works item7"><h1 class="work-name">Multi-Post Stories</h1>
+            <h2 class="work-des">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </h2>
+        <div class="work-lang">
+            <a href="">css</a><a href="">html</a><a href="">bootstrap</a><a href="">ruby</a>
+        </div>
```

Comment:
- [ ] he list of project technologies (e.g., "JavaScript", "HTML" and "CSS") are a list ul and each of them a li, kindly refactor with `ul li` not a tag